src/components/network-status/NetworkStatusMessage.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react'
+
+import { useNetworkStatus } from './useNetworkStatus'
+import './styles.css'
+
+export const OFFLINE_MESSAGE = 'you are working in offline mode'
+export const ONLINE_MESSAGE = 'you are back online'
```

Comment:
these messages will need to be internationalized. no need to get all of the translations, but let's reference this as a key and then put these messages in the `locales` folder

Reply:
Ok, I'll do that